lib/bloc/account/account_actions.dart

```diff
@@ -46,4 +46,11 @@ class ChangeSyncUIState extends AsyncAction {
   ChangeSyncUIState(this.nextState);
 }
 
-class FetchRates extends AsyncAction {}
\ No newline at end of file
+class FetchRates extends AsyncAction {}
+
+class UpdateSecurityModel extends AsyncAction {
```

Comment:
This action is related to UserProfileBloc, therefore should be moved under the user_profile folder.

Reply:
👍 
On top of this I suggest we separate AsyncAction from account_actions and import it in both account & user actions.